src/uniparc/components/results/__tests__/__snapshots__/UniParcCard.spec.tsx.snap

```diff
@@ -13,7 +13,7 @@ exports[`UniRefCard tests should render and match snapshot 1`] = `
       <div
         class="card__content"
       >
-        <section
+        <div
```

Comment:
The ` · ` disappears for some reason.
![image](https://user-images.githubusercontent.com/4357962/112285371-289dcf00-8c82-11eb-9027-4ec02114a486.png)

Reply:
This was relying on classes defined in the stylesheet for `RenderColumnsInCard` so now that they're in CSS modules I needed to explicitly import them. Done now